t/op/each.t

```diff
@@ -127,14 +127,14 @@ SKIP: {
 my @tests = (&next_test, &next_test, &next_test);
 {
     package Obj;
-    sub DESTROY { print "ok $::tests[1] # DESTROY called\n"; }
+    sub DESTROY { print "ok $tests[1] # DESTROY called\n"; }
```

Comment:
this seems incorrect as before it was using tests from `main::` and now it's using it from `Obj::`

I wonder if `my @tests` needs to become `our @tests`?

Reply:
> this seems incorrect as before it was using tests from `main::` and now it's using it from `Obj::`
> 

I don't think so.  Aren't they using `@tests` as defined and assigned to on line 127?

Reply:
you are correct, thanks